var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -82,7 +82,7 @@ class Xsdk(Package):
     depends_on('sundials@3.1.0~int64+hypre', when='@0.3.0')
     depends_on('sundials@3.1.0~int64+hypre', when='@develop')
 
-    depends_on('plasma@17.1', when='@develop %gcc@4.9:')
+    depends_on('plasma@develop', when='@17.2: %gcc@6.0:')
```

Comment:
perhaps you meant to to change `plasma@develop` into `plasma@17.2:` instead of changing the `when=` part.

Reply:
ops fixed now